kpatch-build/kpatch-build

```diff
@@ -304,6 +311,7 @@ find_special_section_data() {
 	[[ -z "$EX_STRUCT_SIZE" ]]  && die "can't find special struct paravirt_patch_site size"
 	[[ -z "$PARA_STRUCT_SIZE" && "$CONFIG_PARAVIRT" -ne 0 ]] && die "can't find special struct paravirt_patch_site size"
 	[[ -z "$ORC_STRUCT_SIZE" && "$CONFIG_UNWINDER_ORC" -ne 0 ]] && die "can't find special struct orc_entry size"
+	[[ -z "$JUMP_STRUCT_SIZE" ]] && die "can't find special struct jump_entry size"
```

Comment:
Not a big deal since we generally target distro kernels, but I believe this introduces a new configuration requirement of `CONFIG_JUMP_LABEL=y`.  The commit looks fine, I just wanted to highlight this in case you might want to make it optional.

Reply:
Good point

Reply:
Happy to approve/merge if you like it as is... alternatively we can make it optional or document it somewhere (though it's not like we've collected up and doc'd all the other CONFIG_ requirements to date.)